js/tiles.js

```diff
@@ -177,3 +177,11 @@ const door = {
     },
   })
```

Comment:
please delete this and the following line, so you are inside the door variable.

Reply:
or maybe I was wrong... The idea is to have your tile available as "door.black" in JavaScript...

Reply:
Sure!

Reply:
When you did this, you can edit lavels.js to include the tile somewhere... Then you can navigate there or start there.